readme.md

```diff
@@ -123,6 +123,7 @@ As defined by Wikipedia a shim is a small library that transparently intercepts
 - [Chronoscope World](https://mprove.de/chronoscope/world.html) – A viewer for georeferenced IIIF maps. It is also a fast viewer for browsing IIIF books.
 - [Clover IIIF](https://samvera-labs.github.io/clover-iiif/) - IIIF Presentation API Manifest viewer handling Image, Sound, and Video canvases for React.js
 - [Diva.js](https://ddmal.github.io/diva.js/) - IIIF image viewer optimized for speed and flexibility.
+- [Glycerine Viewer](https://github.com/Systemik-Solutions/glycerine-viewer) - An elegant a contemporary Vue 3 Viewer with a comprehensive annotation feature set.
```

Comment:
Should this be "An elegant and contemporary viewer"?

Reply:
agree